frappe/database/query.py

```diff
@@ -283,6 +284,36 @@ def add_conditions(self, conditions: frappe.qb, **kwargs):
 
 		return conditions
 
+	@staticmethod
+	def get_condition_from_nested_sets(value: list | tuple, table: str):
```

Comment:
```suggestion
	def get_nested_set_hierarchy_result(value: list | tuple, table: str):
```

also why is this a `staticmethod`? 🤔

Reply:
didn't feel like this function should be a part of this class since it uses qb directly to generate queries 😢

Reply:
done 👍🏿